src/40select.js

```diff
@@ -447,40 +304,50 @@ function modify(query, res) {
 
 		case 'COLUMN':
 			if (res.length === 0) return [];
-
-			let key;
-			if (columns && columns.length > 0) {
-				key = columns[0].columnid;
-			} else {
-				key = Object.keys(res[0])[0];
-			}
-
-			let ar = [];
-			for (var i = 0, ilen = res.length; i < ilen; i++) {
-				ar.push(res[i][key]);
-			}
-
-			return ar;
+			const key = columns?.[0]?.columnid || Object.keys(res[0])[0];
+			return res.map(r => r[key]);
```

Comment:
Nice with an improvement. But not really related to the PR. In the future please make a seperate PR with things not related to the PR. Lets just keep it for now.